emote/memory/table.py

```diff
@@ -70,6 +70,11 @@ def __init__(
 
         self.clear()
 
+    def resize(self, new_size):
+        with self._lock:
+            if new_size > self._maxlen:
+                self._maxlen = new_size
```

Comment:
I think this should either raise a value error or shrink the memory if it's reduced. Not just silently ignore it.